extra/linux/redhat/alacritty.spec

```diff
@@ -7,14 +7,16 @@ URL:           https://github.com/alacritty/alacritty
 VCS:           https://github.com/alacritty/alacritty.git
 Source:        alacritty-%{version}.tar
 
-BuildRequires: rust >= 1.41.0
+BuildRequires: rust >= 1.43.0
 BuildRequires: cargo
 BuildRequires: cmake
-BuildRequires: freetype-devel
-BuildRequires: fontconfig-devel
-BuildRequires: libxcb-devel
+BuildRequires: gcc-c++
+BuildRequires: python3
+BuildRequires: pkgconfig(freetype2)
+BuildRequires: pkgconfig(fontconfig)
+BuildRequires: pkgconfig(xcb)
 BuildRequires: desktop-file-utils
-BuildRequires: python36
+BuildRequires: ncurses
```

Comment:
Does Fedora not have a "base" package that is always expected to be installed with things like ncurses? Not much you can do on linux without ncurses installed.

Reply:
For F32 it looks like it was pulled in by the coreutils package, for F33 that package was updated and ncurses was made optional so it needed to be added to the build requires. 

ref: https://src.fedoraproject.org/rpms/coreutils/c/5d1216301214534571714883927c09b5771fcf7d?branch=master